homeassistant/components/homewizard/button.py

```diff
@@ -54,4 +57,10 @@ def __init__(
 
     async def async_press(self) -> None:
         """Identify the device."""
-        await self.coordinator.api.identify()
+        try:
+            await self.coordinator.api.identify()
+        except RequestError as ex:
+            raise HomeAssistantError from ex
+        except DisabledError as ex:
+            await self.hass.config_entries.async_reload(self.coordinator.entry_id)
```

Comment:
I'm not sure what this will do?
What is the reload going to address?

Reply:
The API was disabled by the user so HA can't communicate with the device anymore. A reload triggers the reauth flow in init.

Reply:
👍  oki!